frontend/src/__tests__/cypress/cypress.config.ts

```diff
@@ -135,7 +135,11 @@ export default defineConfig({
         await mergeFiles(outputFile, inputFiles);
       });
 
-      return config;
+      // Apply retries only for tests in the "e2e" folder
+      return {
+        ...config,
+        retries: config.specPattern.includes('/e2e/') ? 2 : config.retries,
```

Comment:
Our e2e is run when `const e2e = !env.CY_MOCK && !env.CY_RECORD`. Either create a new `const` or inline as follows:
```suggestion
        retries: !env.CY_MOCK && !env.CY_RECORD ? 2 : config.retries,
```

Do we want to include retries in open mode? If not, you can configure open and run mode separately:
See https://docs.cypress.io/app/guides/test-retries#Global-Configuration
```suggestion
        retries: !env.CY_MOCK && !env.CY_RECORD ? { runMode: 2, openMode: 0 } : config.retries,
```